package.json

```diff
@@ -11,16 +11,18 @@
     "frontend",
     "backend"
   ],
+  "engines": {
+    "node": ">= v18.17.0"
+  },
   "scripts": {
     "environment": "ts-node --skipProject ./scripts/genesisEnvironmentVariables.ts",
-    "build:client": "cd ./frontend && yarn build",
-    "build:well-known": "ts-node --skipProject ./scripts/genesisWellKnownDidConfig.ts",
-    "build": "yarn build:well-known && yarn build:client",
-    "start:client": "ts-node ./scripts/updatePorts.ts && cd ./frontend && yarn start",
-    "start:server": "cd ./backend && yarn start",
+    "well-known-config": "ts-node --skipProject ./scripts/genesisWellKnownDidConfig.ts",
+    "build:frontend": "cd ./frontend && yarn build",
+    "build:server": "cd ./backend & yarn check",
+    "build": "yarn build:frontend && yarn build:server",
+    "start:frontend": "ts-node ./scripts/updatePorts.ts && cd ./frontend && yarn start",
+    "start:server": "cd ./backend && yarn dev",
```

Comment:
Isn't always the pairs: 
- either `frontend` and `backend`
-  or `server` and `client`

Reply:
[d48308](https://github.com/KILTprotocol/web3-login-demo/pull/35/commits/5d48308e402818e595f47deea15ee44c6dc69b66)